python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_selector_group_chat.py

```diff
@@ -184,17 +206,34 @@ async def select_speaker(self, thread: List[BaseAgentEvent | BaseChatMessage]) -
 
         # Select the next speaker.
         if len(participants) > 1:
-            agent_name = await self._select_speaker(roles, participants, history, self._max_selector_attempts)
+            agent_name = await self._select_speaker(roles, participants, self._max_selector_attempts)
         else:
             agent_name = participants[0]
         self._previous_speaker = agent_name
         trace_logger.debug(f"Selected speaker: {agent_name}")
         return agent_name
 
-    async def _select_speaker(self, roles: str, participants: List[str], history: str, max_attempts: int) -> str:
+    def construct_message_history(self, message_history: Sequence[LLMMessage]) -> str:
```

Comment:
```suggestion
    def construct_message_history(self, message_history: List[LLMMessage]) -> str:
```

Make sure it plays nicely with the ChatCompletionContext